css/style.css

```diff
@@ -25,15 +25,15 @@
         size of the image-caption */
         #caption {
             font-size: 17px;
-            font-family: Gill Sans;
+            font-family:sans-serif
             color: black;
         }
 
         /* Styling the content of the page like-
         padding, font-size, font color etc.*/
-        div#tribute-data {
+        #tribute-data {
             background-color: rgb(46, 139, 87, 0.25);
-            box-shadow: 20px 20px 20px #98FB98;
+            box-shadow: 20px 20px 20px #b0f1b0;
```

Comment:
Kindly add them together and also include those color codes(#98FB98 and b0f1b0).